music21/midi/translate.py

```diff
@@ -522,25 +583,35 @@ def midiEventsToChord(eventList, ticksPerQuarter=None, inputM21=None):
         <music21.midi.MidiEvent NOTE_ON, track=1, channel=None, pitch=46, velocity=94>]
 
     Changed in v.7 -- Uses the last DeltaTime in the list to get the end time.
+    Changed in v.7.3 -- Returns a :class:`~music21.percussion.PercussionChord` if
+    any event is on channel 10. Returns None if `inputM21` provided.
+
+    >>> me2.channel = 10
+    >>> midi.translate.midiEventsToChord([dt1, me1, dt2, me2, dt3, me3, dt4, me4])
+    <music21.percussion.PercussionChord unpitched[A2] unpitched[B2]>
```

Comment:
just as a further refinement -- instead of showing display step on PercussionChord it should be a list of instruments:

<music21.percussion.PercussionChord [Bass Drum, Snare]>

or if mixed with notes:

<music21.percussion.PercussionChord [Vibraslap, D4, D8, Snare]>